cypress-tests/cypress/e2e/configs/Payment/Fiservemea.js

```diff
@@ -153,6 +379,248 @@ export const connectorDetails = {
         },
       },
     },
+    MandateSingleUseNo3DSManualCapture: {
+      Configs: {
+        TRIGGER_SKIP: true,
+      },
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "EUR",
+        mandate_data: singleUseMandateData,
+        billing: billingAddress,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_capture",
+        },
+      },
+    },
```

Comment:
since the `setup_future_usage` does a fallback to `on_session` when mandate is not supported, we can start moving this config to `Commons` (refactor commons) such that, instead of expecting the `501` we expect a `2xx` response which can be overriden by connectors if they expect a different status in the response with `TRIGGER_SKIP` set to `true` just because mandate payments shouldn't happen.

yes, this can be considered in a separate pr given the complexity and the work involved.

Reply:
i would want this to be kept unresolved for now. if not asking for much, can you please open an issue on this topic so that it can be picked in future and not left as a tech debt?